<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLineDefine</class>
 <widget class="QDialog" name="DialogLineDefine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>连接定义</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>连接代号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdWireTag">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>4</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>40</y>
     <width>221</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>导线属性</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>70</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>颜色/编号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CbColor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>12</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>棕色 BN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>红色 RD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黑色 BK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>橙色 OG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黄色 YE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>绿色 GN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>蓝色 BU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>紫色 VT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>灰色 GY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>白色 WH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>粉红色 PK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>金色 GD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>青绿色 TQ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>银色 SR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>绿色/黄色 GNYE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>屏蔽 SH</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>70</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>截面积/直径</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CbDiameter">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>38</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.34</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="EdPartCode">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>64</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>物料编码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdType">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>96</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>型    号</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="LbWireAttrView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>21</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnChangeForm">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>167</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>167</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CbTermlShortage">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设置为端子排短接片</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
